### ISSUE TITLE
Terraforged Incompatibility Ongoing 

### ISSUE THREAD

**Issue by Jollyolsport** [nekomaster1000/Infernal-Expansion#250]
Terraforged prevents biomes from generating when used as a world generator.

**Version Information:**
 - Operating System: windows 10
 - Mod Version: infernal expansion 2.1.1, terraforged 1.16.5-0.2.14
 - Minecraft Version: 1.16.5
 - Forge Version: 36.2.0

I have BOP installed as well and went ahead and changed those configs and set their generation to false; Biomes still dont generate

**Comment by Dariensg** [comment]
Have you gone into the TerraForged "Customize" settings before creating the world and changed the Nether generation to `infernalexp:compat_world_type`. If not, you're not correctly doing TerraForged compat and that's your problem. I've tested it and it works with TerraForged, BoP, and BYG all at the same time and the BoP config doesn't even need to be changed for it as long as you do exactly what I just explained.
